akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1349,8 +1337,8 @@ private[cluster] final class FirstSeedNodeProcess(seedNodes: immutable.IndexedSe
  *
  */
 private[cluster] final class JoinSeedNodeProcess(seedNodes: immutable.IndexedSeq[Address]) extends Actor with ActorLogging {
-  import InternalClusterAction._
   import ClusterUserAction.JoinTo
+  import InternalClusterAction._
```

Comment:
I guess you could just `git checkout wip-multi-dc-dev --  akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala` to avoid changes in this file in this PR? Not mendatory but would be nice.